python/lsst/ap/association/dia_collection.py

```diff
@@ -22,9 +22,13 @@
 
 from __future__ import absolute_import, division, print_function
 
-from scipy.spatial import cKDtree
+import numpy as np
+from scipy.spatial import cKDTree
 
-import .dia_object
+import lsst.afw.table as afwTable
+import lsst.pipe.base as pipeBase
+
+from .dia_object import *
```

Comment:
Usually we don't allow wildcard imports into a non-`__init__`.py.

Reply:
Also, I don't quite know what you mean be "setup.cfg", Tim. Is this a file I should add to the repository somewhere? If so is there an example you can link me to, please?

Reply:
@morriscb your reply is to the wrong comment of @timj : I think you meant to reply to his comment just above, re: adding setup.cfg.

Unfortunately, @timj it looks like DM-10493 was closed Won't Fix, so I don't know where the documentation for configuring auto-flake8 actually lives...

Reply:
It would be great if you could enable flake8 testing in this package. You do that by copying a `setup.cfg` from another package (until I update the dev guide). Something like this is fine:
```
[flake8]
max-line-length = 110
ignore = E133, E226, E228, N802, N803, N806
exclude = __init__.py

[tool:pytest]
addopts = --flake8
flake8-ignore = E133 E226 E228 N802 N803 N806
```
Then you enable pytest auto discovery in `tests/SConscript` by adding `pyList=[]` to the `tests()` method.

Reply:
Hi @timj, I tried this and I get "scons: Nothing to be done for `tests'." when using pyList=[] in the tests() method. When I remove this, scons then performs the tests. Could you take a look at the current version that I committed, please, and let me know what is missing? I Copied your text above and put it into setup.cfg in the root directory of the repository.

Reply:
Are you running with an old weekly? You need current master or weekly _35.

Reply:
Thanks @timj, everything works now and comes up "green".